tests/ios/app-clip/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+load("//rules:framework.bzl", "apple_framework")
+load("//rules:app_clip.bzl", "ios_app_clip")
+
+apple_framework(
+    name = "AppClipFramework",
+    srcs = glob(["AppClipFramework/**/*.swift"]),
+    platforms = {"ios": "13.0"},
+    visibility = ["//visibility:public"],
+    sdk_frameworks = [
+        "UIKit",
+        "SwiftUI",
+    ],
+)
+
+ios_app_clip(
```

Comment:
Should we also test for custom infoplists and xcconfigs? Feel free to do in a follow up but it's important to catch when those fail.